packages/shared/postcss.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
           pageMaxWidth: '42.5rem',
           headerHeight: '3rem',
           headerRankHeight: '4.5rem',
-          navBarHeight: '3rem',
+          navBarHeight: '4rem',
```

Comment:
Do note that this will affect everything that uses it in one go.
Not sure it's intended/accepted (if so it's fine, but just make sure nothing accidentally get's affected by this)

Reply:
Is it only 4rem? The screenshot looks like it is missing some vertical padding.

Reply:
This actually did not do anything, I reverted 🙏